lib/auth/oidc.go

```diff
@@ -369,8 +369,15 @@ func checkEmailVerifiedClaim(claims jose.Claims) error {
 	unverifiedErr := trace.AccessDenied("email not verified by OIDC provider")
 
 	emailVerified, hasEmailVerifiedClaim, _ := claims.StringClaim(claimName)
-	if hasEmailVerifiedClaim && emailVerified == "false" {
-		return unverifiedErr
+	if hasEmailVerifiedClaim {
+		if emailVerified == "false" {
```

Comment:
Perhaps we can allow more formats? If so, https://pkg.go.dev/strconv#ParseBool might be a good choice.

Reply:
No need to support more formats. Technically, even "true" is out of spec, but one or two IDPs we know of get it wrong.